src/MOI_wrapper.jl

```diff
@@ -28,7 +28,12 @@ end
 DualOptimizer constructor
 """
 function DualOptimizer(dual_optimizer::OT) where {OT <: MOI.ModelLike}
```

Comment:
The user should call `DualOptimizer{T}` here so that we don't have to hardcode `Float64`. You can have yet another constructor `DualOptimizer(dual_opt) = DualOptimizer{Float64}(dual_opt)` if you want it to be the default